checkers/model_form.py

```diff
@@ -0,0 +1,55 @@
+import ast
+
+from .checker import Checker
+from .issue import Issue
+
+
+class DJ06(Issue):
+    code = 'DJ06'
+    description = 'ModelForm.Meta should not set exclude, set fields instead'
+
+
+class ModelFormChecker(Checker):
+
+    def checker_applies(self, node):
+        for base in node.bases:
+            if self.is_model_form_attribute(base) or self.is_model_form_name(base):
```

Comment:
what about adding a meaningful name to the check?

```python
for base in node.bases:
    is_model_form = self.is_model_form_attribute(base) or self.is_model_form_name(base)
    if is_model_form:
        return True
return False
```

Reply:
fixed 👍